packages/blockchain/ada/src/lib/services/ada.crypto.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  bech32,
+  derivePrivate,
+  derivePublic,
+  getPubKeyBlake2b224Hash,
+  mnemonicToRootKeypair,
+  packBaseAddress,
+  // @ts-ignore
```

Comment:
Why this ts-ignore?

Reply:
throws lint error otherwise

Reply:
yeah, its because of missing types of cardano-crypto.js. Add comment with info there pls.

Reply:
okay